opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -423,7 +423,10 @@ def start_span(
         kind: trace_api.SpanKind = trace_api.SpanKind.INTERNAL,
     ) -> typing.Iterator["Span"]:
         """See `opentelemetry.trace.Tracer.start_span`."""
-        with self.use_span(self.create_span(name, parent, kind)) as span:
+
+        span = self.create_span(name, parent, kind)
```

Comment:
This approach seems weird, not sure if other Python libraries have similar concept of using a ContextManager with an optional flag to avoid clean up (e.g. `using open(filename, 'rb', close_on_exit=False) as f:`).
Is it possible to use the following approach?
```python
span = self.create_span(name, parent, kind)
span.start()
self.use_span(span)
do_something()
span.end()
```

Reply:
Only example I could find was `closefd` in the built-in `open` function (https://docs.python.org/3/library/functions.html#open). But I know in .NET there are similar examples where you choose whether some TextReader should close the underlying file.

See also https://github.com/open-telemetry/opentelemetry-python/issues/97#issuecomment-534048872

Reply:
> Is it possible to use the following approach?

The problem with that is that after `span.end()`, the `span` is still active (which may be what you want if you have async children, but not in the general case).